install_cardio.rb

```diff
@@ -17,7 +17,7 @@
 }
 system('cd Example; carthage update --platform ios')
 
-project_path = 'Example/Stripe iOS Example (Simple).xcodeproj'
+project_path = 'Example/Standard Integration (Simple).xcodeproj'
```

Comment:
```
$ ./install_cardio.rb
▸ Installing card.io – this may take a while
*** Cloning card.io-iOS-source
*** Checking out card.io-iOS-source at "iOS_5.4.1"
*** Checking out Alamofire at "4.4.0"
*** xcodebuild output can be found in /var/folders/w7/d58yzqdn6617q8ytgqvxx8500000gn/T/carthage-xcodebuild.r7Lovx.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "CardIO" in icc.xcodeproj
/Users/joeydong/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/xcodeproj-1.5.0/lib/xcodeproj/project.rb:99:in `open': [Xcodeproj] Unable to open `/Users/joeydong/stripe/stripe-ios/Example/Standard Integration (Simple).xcodeproj` because it doesn't exist. (RuntimeError)
	from ./install_cardio.rb:21:in `<main>'
```
These paths should be `Example/Standard Integration (Swift)`